panel/pane/base.py

```diff
@@ -153,24 +166,28 @@ def __getitem__(self, index):
     #----------------------------------------------------------------
 
     @property
-    def _linkable_params(self):
+    def _linkable_params(self) -> List[str]:
```

Comment:
I just want to note that with the future import, this could be replaced with `def _linkable_params(self) -> list[str]`.